### PR TITLE
PWN-9504 - fixes according to comments

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9504

## Description of Work

PWN-9504 - fixes according to comments

### CODE REVIEW

File: token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt [hunk 1]
```diff
@@ -38,14 +38,15 @@ internal class TokenMetadataRemoteRepository(
...
             api.getZipFile("${tokenServiceUrl}get_all_tokens_info", ifModifiedSince)
-        } catch (e: HttpException) {
-            return@withContext if (e.code() == HttpURLConnection.HTTP_NOT_MODIFIED) {
-                UpdateTokenMetadataResult.NoUpdate
-            } else {
-                UpdateTokenMetadataResult.Error(e)
-            }
         } catch (e: Throwable) {
-            return@withContext UpdateTokenMetadataResult.Error(e)
+            return@withContext when (e) {
```
Comment (gslevinkov) [inline token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt:42]: to a method

Comment (gslevinkov) [reply token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt:42]: and this is very hard to read 
try doin `if (e is HttpException && e.code() == HttpURLConnection.HTTP_NOT_MODIFIED) { NoUpdate } else  UpdateTokenMetadataResult.Error

**Review by gslevinkov — APPROVED** [review]

File: token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt (hunk 1 above)
Comment (kabramovp2p) [reply token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt:42]: zaeba =)

Comment (kabramovp2p) [reply token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt]: `fixed`

File: token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt [hunk 2]
```diff
@@ -63,6 +57,15 @@ internal class TokenMetadataRemoteRepository(
         UpdateTokenMetadataResult.NewMetadata(remoteTokensMetadata = metadata)
     }
 
+    private fun handleError(error: Throwable): UpdateTokenMetadataResult {
+        return when {
```
Comment (gslevinkov) [inline token-service/src/main/java/org/p2p/token/service/repository/metadata/TokenMetadataRemoteRepository.kt:61]: if when has only 2 branches - we use if

**Review by 1sakov — APPROVED** [review]

**Pull request merged**